tests/openvino/test_exporters_cli.py

```diff
@@ -1261,6 +1271,81 @@ def test_exporters_cli_with_default_config(
                     f"Parameter {key} not matched with expected, {model_quantization_config[key].value} != {value}",
                 )
 
+    DEFAULT_IGNORED_SCOPE_TEST_CONFIGURATIONS = [
+        (
+            "speecht5",
+            "microsoft/speecht5_tts",
+            "decoder",
+            AutoModelForTextToSpectrogram,
+            OVModelForSpeechSeq2Seq,
+            '--task text-to-speech --weight-format int8 --model-kwargs \'{"vocoder": "fxmarty/speecht5-hifigan-tiny"}\'',
+        ),
+    ]
+
+    # Filter by supported models, same pattern as for the other test
+    SUPPORTED_DEFAULT_IGNORED_SCOPE_TEST_CONFIGURATIONS = [
+        config
+        for config in DEFAULT_IGNORED_SCOPE_TEST_CONFIGURATIONS
+        if TEST_NAME_TO_MODEL_TYPE.get(config[0], config[0]) in get_supported_model_for_library("transformers")
+    ]
+
+    @parameterized.expand(SUPPORTED_DEFAULT_IGNORED_SCOPE_TEST_CONFIGURATIONS)
+    def test_exporters_cli_with_default_ignored_scope(
+        self,
+        test_model_id,
+        model_id,
+        ov_model_name,
+        auto_model_cls,
+        ov_model_cls,
+        options,
+    ):
+        with TemporaryDirectory() as tmpdir:
+            tmpdir = "tmp"
+            # 1. Create a local copy of the model so that we can override _name_or_path
+            pt_model = auto_model_cls.from_pretrained(MODEL_NAMES[test_model_id])
+            pt_model.save_pretrained(tmpdir)
+            try:
```

Comment:
here you could use instead `maybe_save_preprocessors`  https://github.com/huggingface/optimum/blob/0227a1ce9652b1b02da5a510bf513c585608f8c2/optimum/utils/save_utils.py#L65